models/lido/liquidity/arbitrum/lido_liquidity_arbitrum_curve_pools.sql

```diff
@@ -16,7 +16,7 @@
 {% set project_start_date = '2022-10-06' %}
 
 with dates as (
-select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 day)) as day
+select explode(sequence(date_trunc("day", now() - interval '1 week'), now(), interval 1 day)) as day
```

Comment:
you'll need both filters, rather than replace the old. you can add the `if is incremental` blocks here, add the sql statement for each incremental / not incremental.
for example:
```
{% if not is_incremental() %}
select explode(sequence(to_date('{{ project_start_date }}'), now(), interval 1 day)) as day
{% else %}
select explode(sequence(date_trunc("day", now() - interval '1 week'), now(), interval 1 day)) as day
{% endif %}
```

Reply:
you also don't need to default to the one week for incremental runs, it can be just one day back if that works too

Reply:
thanks @jeff-dude 
fixed filters
it seems ok 1 week currently